Makefile

```diff
@@ -18,7 +18,11 @@
 
 all: binaries
 
-CFLAGS = -std=c99 -Wall -O3 -g -D_GNU_SOURCE -DNO_LIBNUMA
+CFLAGS := -std=c99 -Wall -O3 -g -D_GNU_SOURCE -DNO_LIBNUMA -DNDEBUG=1
+
+ifdef WITH_TCPDIRECT
+	CFLAGS += -DWITH_TCPDIRECT
```

Comment:
```suggestion
	CFLAGS += -DWITH_TCPDEVEMEM
```

Reply:
Also, in the UDMABUF-only support, you may want

WITH_TCPDEVMEM_CUDA
WITH_TCPDEVMEM_UDMABUF